main.go

```diff
@@ -370,13 +369,14 @@ func runCLIApp(c *cli.Context) (err error) {
 	// daemonize gives us and telling it about the outcome.
 	var mfs *fuse.MountedFileSystem
 	{
-		mountStatus := logger.NewNotice("")
+		mountStatus := logger.NewInfo("")
 		mfs, err = mountWithArgs(bucketName, mountPoint, flags, mountStatus)
 
 		if err == nil {
 			mountStatus.Println("File system has been successfully mounted.")
 			daemonize.SignalOutcome(nil)
 		} else {
+			mountStatus.Printf("Error while mounting gcsfuse: %v\n", err)
```

Comment:
add a note here that printing via mountstatus will have duplicate logs on the console when run in foreground without log file. But this is important to avoid losing error logs when run in background mode.

Reply:
done.